apps/server/src/controllers/ontimeController.validate.ts

```diff
@@ -63,6 +63,7 @@ export const validateSettings = [
   body('operatorKey').isString().isLength({ min: 0, max: 4 }).optional({ nullable: true }),
   body('timeFormat').isString().isIn(['12', '24']),
   body('language').isString(),
+  body('serverPort').isNumeric().optional(),
```

Comment:
it seems that there is a builtin validator for this
https://express-validator.github.io/docs/api/validation-chain/#isport

This should also make sure in is a valid range